style.css

```diff
@@ -0,0 +1,70 @@
+html,
+body {
+  margin: 0;
+  padding: 0;
+}
+
+.nav-bar {
+  display: flex;
```

Comment:
### Optional
- [ ] It would be better if we implement the CSS DRY principle, for instance; `display: flex` has been repeated, let us kindly fix it
- [ ] Going forward, let us format our PR nicely with header and bullet points explaining what we did